Menu/Menu.js

```diff
@@ -1,10 +1,17 @@
-
 const toggleMenu = () => {
-  // Toggle the "menu--open" class on your menu refence. 
-}
+  // Toggle the "menu--open" class on your menu refence.
+  // menu.classList.toggle('menu--open');
 
+  if (menu.style.width === '350px') {
+    TweenMax.to(menu, 0.5, { width: 0, ease: Power1.easeOut });
+  } else {
+    TweenMax.to(menu, 0.5, { width: 350, ease: Power1.easeOut });
+  }
+};
```

Comment:
Personally I would use transform on this to just move the menu off the side of the page when closed:

```transform: translateX(100%)```

This prevents the text from getting squashed as it closes.